Mac/MainWindow/OPML/ExportOPMLWindowController.swift

```diff
@@ -75,7 +75,7 @@ import Account
 	func exportOPML(account: Account) {
 		
 		let panel = NSSavePanel()
-		panel.allowedFileTypes = ["opml"]
+		panel.allowedContentTypes = [.xml]
```

Comment:
Should this be `.opml` instead of `.xml`?

Reply:
I've addressed this in 7a008db — `allowedContentTypes` is `.opml` and, to ensure it is used, the Save panel doesn't provide an extension in `nameFieldStringValue` as the system provides it from the `allowedContentTypes` array.